console-subscriber/src/lib.rs

```diff
@@ -922,7 +923,42 @@ impl Server {
     /// ```
     /// [`serve_with`]: Server::serve_with
     pub async fn serve(self) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-        self.serve_with(tonic::transport::Server::default()).await
+        self.serve_with(tonic::transport::Server::builder()).await
```

Comment:
I don't understand the reason for changing this line. Can we keep it the same?

If not, we should update the documentation as well.

Reply:
I accidentally changed it. I've reverted it. Thanks!